mlflow/server/js/src/components/HomeView.js

```diff
@@ -59,7 +59,6 @@ class HomeView extends Component {
              <NoExperimentView/>
             }
           </div>
-          <div className="experiment-view-right"/>
```

Comment:
Not needed?

Reply:
yep, this element just existed to take up 10% of the screen width on the right side :P - we now take up the whole width minus 64px on each side (hence the magic 128px showing up everywhere)